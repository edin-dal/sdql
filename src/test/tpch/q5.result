<INDONESIA,566380>:1
<CHINA,740211>:1
<VIETNAM,1.00093e+06>:1
<INDIA,422875>:1
<JAPAN,660651>:1