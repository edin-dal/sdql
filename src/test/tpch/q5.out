INDONESIA:566380
VIETNAM:1.00093e+06
JAPAN:660651
CHINA:740211
INDIA:422875